packages/@aws-cdk/aws-codepipeline/test/test.cloudformation-pipeline-actions.ts

```diff
@@ -63,6 +63,7 @@ export = {
       changeSetName,
       role: changeSetExecRole,
       templatePath: new ArtifactPath(buildAction.artifact!, 'template.yaml'),
+      templateConfiguration: new ArtifactPath(buildAction.artifact!, 'templateConfig.json')
```

Comment:
Is `templateConfiguration` always supposed to be an artifact path? Why not just change it's type to `ArtifactPath` and dereference `.location` in the L2?

Reply:
As far as I'm aware - yes. Will try that instead.

However and maybe I'm being silly - would we ever not want to do this?

Reply:
You mean implement `resolve` - ideally no... It's a cheap trick that we are trying to avoid at the L2 surface area